spring-social-zotero/src/main/java/org/springframework/social/zotero/api/impl/GroupsTemplate.java

```diff
@@ -247,6 +249,55 @@ public void updateItem(String groupId, Item item, List<String> ignoreFields, Lis
             throw new ZoteroConnectionException("Could not update item.", e);
         }
     }
+    
+    
+    
+    /**
+     * This method makes a batch request call to Zotero to update items
+     * 
+     * @param groupId      group id of citations
+     * @param items        items that have to be updated
+     * @param ignoreFields fields that are not necessary while updating citations
+     * 
+     * @return ItemCreationResponse returns response from Zotero
+     */
+
+    /*
+     * The response returned by Zotero while creating a single item or updating
+     * multiple items is in the same format. So, we can use "ItemCreationResponse" data
+     * type to capture the response in both cases.
+     */
+    @Override
+    public ItemCreationResponse batchUpdateItems(String groupId, List<Item> items, List<String> ignoreFields)
```

Comment:
javadoc should be right before method; comments for implementation should be in the code where relevant.